docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md

```diff
@@ -102,6 +102,7 @@ The following toggles require explicitly setting Grafana's [app mode]({{< relref
 | -------------------------------------- | --------------------------------------------------------- |
 | `dashboardPreviewsAdmin`               | Manage the dashboard previews crawler process from the UI |
 | `showFeatureFlagsInUI`                 | Show feature flags in the settings UI                     |
+| `publicDashboardsEmailSharing`         | Public dashboard email sharing                            |
```

Comment:
I'm curious, what is the purpose of this setting? I wonder if the description can provide a bit more information. Currently, it simply repeats the name of the toggle.

Reply:
The feature lets you only share a public dashboard to specific emails. 

Maybe something like `Allows public dashboard sharing to be restricted to only allowed emails` ?